core/common/src/main/java/alluxio/util/UnderFileSystemUtils.java

```diff
@@ -158,5 +160,14 @@ public static boolean isSwift(UnderFileSystem ufs) {
     return "swift".equals(ufs.getUnderFSType());
   }
 
+  /**
+   * @param uri the UFS path
+   * @return the bucket or container name of the object storage
+   */
+  public static String getBucketName(AlluxioURI uri) {
+    Preconditions.checkState(isObjectStorage(uri.toString()));
+    return uri.getAuthority();
```

Comment:
I don't have any good suggestions immediately and I think the issue is deeper than just the method naming. I wasn't looking for any additional changes to address this comment in this PR.